src/redux/store.ts

```diff
@@ -10,7 +10,16 @@ const store = configureStore({
   middleware: (getDefaultMiddleware) =>
     getDefaultMiddleware({
       serializableCheck: {
-        ignoredActions: ['counter/addModelsToStorage'],
+        // Ignore these action types
```

Comment:
can we connect to discuss this block of changes? 🙏

Reply:
Sure